integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA4883SynapseHandlerBlockingCallTestCase.java

```diff
@@ -43,8 +43,6 @@
     public void init() throws Exception {
```

Comment:
License needs to be updated with the year created and the license format.

Reply:
It's already there :)